lib/includes/affiliate-manager.php

```diff
@@ -8,7 +8,7 @@
 $aff_name  = get_term_meta( $aff_id, 'name', true );
 $aff_image = wp_get_attachment_image( get_term_meta( $aff_id, 'picture', true ), 'person_thumbnail', false, array( 'class' => 'AffiliateManager__image' ) );
 if ( ! isset( $aff_image ) || '' === $aff_image ) {
-	$aff_image = '<img class="AffiliateManager__image" src="' . esc_url( get_template_directory_uri() ) . '/assets/images/affiliate_manager_avatar.svg" alt="' . $aff_name . '" />';
+	$aff_image = get_avatar( get_term_meta( $aff_id, 'email', true ), $size = '145', $default = "' . esc_url( get_template_directory_uri() ) . '/assets/images/affiliate_manager_avatar.svg'" );
```

Comment:
Aren't we going to use a custom CSS class? you should be able to add it as last parameter do get_avatar as array( class=> array('class1', 'class2'))

Reply:
added, but it was needed to add an empty argument before it, otherwise it would be considered as alt, not class

Reply:
yeah, it's possible